src/per/i2c.cpp

```diff
@@ -47,11 +54,12 @@ class I2CHandle::Impl
     // scheduling and global functions
     struct DmaJob
     {
-        uint16_t                       slave_address    = 0;
+        uint16_t                       slave_address    = 0x10;
```

Comment:
what's the reason for this change?

Reply:
0 is a reserved address for I2C (as are 1-7 and 119-127), so 16 is a safer default address. Not a big change, and I could revert it, but it feels a bit more appropriate this way.